doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -590,15 +590,31 @@ Printing universes
    terms apparently identical but internally different in the Calculus of Inductive
    Constructions.
 
-.. cmd:: Print {? Sorted } Universes {? Subgraph ( {* @qualid } ) } {? @string }
+.. cmd:: Print {? Sorted } Universes {? Subgraph ( {* @debug_univ_name } ) } {? @string }
```

Comment:
Why not `univ_name`, without the `debug_` prefix.

Reply:
It will not be used in other places which take universe names, like Constraint or in terms

Reply:
Then, maybe `qualified_univ_name`?

And why not using these names elsewhere too?

Alternatively, what about conventionally calling these universes `u1` rather than `1`?

Reply:
The numbers in raw universes are extremely unstable, they should never be used.

Reply:
By the way, from where will the user know the universe number? E.g., I don't see numbers in the following cases:
```coq
Definition a := Type.
Set Printing Universes.
Print a.
(* a = Type@{a.u0} : Type@{a.u0+1} *)
```
or
```coq
Print Universes.
(* ... Set < a.u0 ... *)
```

Reply:
As a digression, if I do:
```coq
Definition a@{u} := Type.
```
I get `Error: Universe Top.5 (Toplevel input, ...) is unbound.` while I guess we could get a message a (slighty) more canonical message of the form `Error: Universe Top.a.1` or something like that.

Reply:
>By the way, from where will the user know the universe number?

Some universes still have only raw names although https://github.com/coq/coq/pull/19599 tries to make it less common

Reply:
The numbers are for template polymorphism IIUC.
The use case for `Print Universes Subgraph` (at least in my small experience) is when you `Print Universes` in an actual development, the result is absolutely unreadable (and takes forever to print) so you need `Print Universes Subgraph` to know if there is a constraint between two universes somewhere in the (transitive closure of) the unreadable thing. It's a purely debug command.

Reply:
The numbers have nothing to do with template polymorphism.

Reply:
After #/19599, what would be the typical use case for these debug-level universe names?

Reply:
They would still appear when universe names are not generated. Also currently there is no way to get universe names out of module types.

>In particular, the PR seems so "debug"-like that, if documented, it might make more sense to document it in dev/doc/debugging.md than in the reference manual.

No, this PR is for debugging of universe issues in user scripts, not just for coq development debugging.

Reply:
OK. And, indeed, the sentence "The integer in raw universe expressions is extremely unstable, so raw universe expressions should not be used outside debugging sessions." seems enough to clarify the purpose.

(I would remove "extremely" which adds nothing, though)

An example might be worth too.

Reply:
I think it's worth having some emphasis on how unstable it is.

Reply:
It is dependent on the context but still deterministic from a session to the other, so not so unstable either.

Reply:
Then ok for keeping the emphasis.

(This being said, we can argue that the unstability of counters by backtracking is a bug.)